daemon/src/daemon.rs

```diff
@@ -85,9 +83,8 @@ type DocChangedReceiver = broadcast::Receiver<()>;
 pub struct DocumentActor {
     doc_message_rx: mpsc::Receiver<DocMessage>,
     doc_changed_ping_tx: DocChangedSender,
-    editor_clients: HashMap<EditorId, EditorHandle>,
-    /// There's one OTServer per buffer. Same file in a different editor is a different buffer.
-    ot_servers: HashMap<EditorId, HashMap<String, OTServer>>,
+    editor_handles: HashMap<EditorId, EditorHandle>,
+    editor_connections: HashMap<EditorId, EditorConnection>,
```

Comment:
I'm wondering if we can summarize these two as one kind of entity inside here, such that we're forced to keep then in sync?